.github/workflows/deploy-preview.yml

```
@@ -87,3 +104,4 @@ jobs:
                       });
               env:
                   DEPLOY_URL: ${{ steps.deploy.outputs.DEPLOY_URL }}
+                  PR_DETAILS: ${{ steps.pr-details.outputs.result }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)